enigma-tools-u/src/web3_utils/w3utils.rs

```diff
@@ -74,6 +74,11 @@ pub fn connect(url: &str) -> Result<(web3::transports::EventLoopHandle, Web3<Htt
     Ok((_eloop, w3))
 }
 
+pub fn connect_batch(http: Http) -> Web3<Batch<Http>> {
+    let w3 = web3::Web3::new(web3::transports::Batch::new(http));
```

Comment:
```suggestion
    web3::Web3::new(web3::transports::Batch::new(http))
```
You can return this directly

Reply:
You're right. I'm not using the batch transport anymore. I will clean-up.

Reply:
Removed `connect_batch` in commit b9f50bc39689924bc12c3f0e14ee5792b72c861d.